compiler/codegen/symbol.cpp

```diff
@@ -1652,7 +1672,7 @@ GenRet FnSymbol::codegen() {
 }
 
 // Supports the creation of a python module with --library-python
-void FnSymbol::codegenPXD() {
+void FnSymbol::codegenPython(bool pxd) {
```

Comment:
bool -> enum please
also I'd prefer this just be named codegenPythonPXDPYD e.g. but I can live with codegenPython